plugin.video.videodevil/resources/catcher.list

```diff
@@ -102,17 +102,20 @@ build=%s|Referer=https://www.youporn.com/
 quality=standard
 ########################################################
 title=extremetube.com
-target=quality_480p":"(http[^"]+)
+target=quality_(?:180|240)p":"([^"]+)
 url=%s
 header=Cookie|age_verified=1
-quality=standard
-target=quality_720p":"(http[^"]+)
+action=prefix
+quality=low
+target=quality_(?:360|480)p":"([^"]+)
 url=%s
 header=Cookie|age_verified=1
-quality=high
-target=quality_1080p":"(http[^"]+)
+action=prefix
+quality=standard
+target=quality_(?:720|1080)p":"([^"]+)

```

Comment:
Have 720 and 1080 as two separate targets please. Otherwise if both exist we will always get the 720 version

Reply:
Will correct that and resubmit